docs/working-copy.md

```diff
@@ -12,12 +12,16 @@ working-copy contents when they have changed. Most `jj` commands you run will
 commit the working-copy changes if they have changed. The resulting revision
 will replace the previous working-copy revision.
 
-Also unlike most other VCSs, added files are implicitly tracked. That means that
-if you add a new file to the working copy, it will be automatically committed
-once you run e.g. `jj st`. Similarly, if you remove a file from the working
-copy, it will implicitly be untracked. To untrack a file while keeping it in
-the working copy, first make sure it's [ignored](#ignored-files) and then run
-`jj file untrack <path>`.
+Also unlike most other VCSs, added files are implicitly tracked by default. That
+means that if you add a new file to the working copy, it will be automatically
+committed once you run e.g. `jj st`. Similarly, if you remove a file from the
+working copy, it will implicitly be untracked. The `snapshot.auto-track` config
+option controls which paths get automatically tracked when they're added. See
```

Comment:
(Optional) I would put a paragraph break before the sentence about `snapshot.auto-track`. If we'd like to connect it to the previous paragraph, perhaps we can split it: "The default can be changed with the `snapshot.auto-track` option. This option controls...`

I'm also wondering how to make this more correct in regards to ignore files. Perhaps just add a sentence: "Files with paths matching [ignore files] are never tracked automatically".

We could also add more details to the ignored files section. Something like:

> Ignored files are never tracked automatically (regardless of the value of `snapshot.auto-track`), but they can still end up being tracked for a few reasons:
>
> - if they were tracked in the parent commit
> - if they did not match the `.gitignore` previously, but the ignore file changed since
> - because of an explicit `jj file track` command
>
> You can untrack such files with the `jj file untrack` command.

(This might benefit from further editing)

Reply:
Makes sense. I dropped the middle bullet because it doesn't sound correct to me (they won't be tracked regardless of the value of `snapshot.auto-track`), and because it's already covered by "Ignored files" at the start of the paragraph.